packages/graphql-client/src/graphql-client.js

```diff
@@ -89,16 +89,16 @@ export class GraphQLClient {
     return await this.mutate(verifyEmailMutation, 'verifyEmail', { token });
   }
 
-  async resetPassword(token: string, newPassword: string): Promise<void> {
+  async resetPassword(token: string, newPassword: any): Promise<void> {
     return await this.mutate(resetPasswordMutation, 'resetPassword', { token, newPassword });
   }
 
-  async sendVerificationEmail(userId: string, email: string): Promise<void> {
-    return await this.mutate(sendVerificationEmailMutation, 'sendVerificationEmail', { userId, email });
+  async sendVerificationEmail(email: string): Promise<void> {
+    return await this.mutate(sendVerificationEmailMutation, 'sendVerificationEmail', { email });
   }
 
-  async sendResetPasswordEmail(userId: string, email: string): Promise<void> {
-    return await this.mutate(sendResetPasswordEmailMutation, 'sendResetPasswordEmail', { userId, email });
+  async sendResetPasswordEmail(email: string): Promise<void> {
```

Comment:
I think we should change the email field to email or username or userId. It will allow different implementations here (admin console, batch reset and so on). It should be same input as the login I think.